lib/ObjWriter/objwriter.cpp

```diff
@@ -550,22 +547,119 @@ extern "C" void EmitDebugFileInfo(ObjectWriter *OW, int FileId,
   OST.EmitCVFileDirective(FileId, FileName);
 }
 
-// This should be invoked at the end of function code emission to flush
-// debug function info.
-extern "C" void EmitDebugFunctionInfo(ObjectWriter *OW,
-                                      const char *FunctionName,
-                                      int FunctionSize) {
+static void EmitPDBDebugVarInfo(MCObjectStreamer &OST, const MCSymbol *Fn,
```

Comment:
Can you change "EmitPDBDebugVarInfo" to "EmitCVDebugVarInfo"? Since we're not in PDB yet. 
Ideally I like it to be an instance method of ObjectWriter object instead of this static function, but that part we can follow up later all together as well.